chapters/curate-maintain/content/reading/README.md

```diff
@@ -1,25 +1,38 @@
 ### Content
 
-Owner: Adrian Șendroiu
-Reviewer: Ștefan Jumărea
+The content maintainer is responsible for overseeing different aspects of the open educational resource, like content management, accepting and reviewing contributions, or community building.
 
-the central part is content management, around a repository
+It is important for such a resource to clearly identify who the maintainer is, for the purpose of transparency and good communication.
```

Comment:
What exactly does `such a resource` refer to?

Reply:
An open education resource.

Reply:
Got it, it's fine.